src/lib/apis/folder.ts

```
@@ -2,8 +2,8 @@ import { type Folder, type MaximFolderResponse, type MaximFoldersResponse } from
 import { MaximAPI } from "./maxim";
 
 export class MaximFolderAPI extends MaximAPI {
-	constructor(baseUrl: string, apiKey: string) {
-		super(baseUrl, apiKey);
+	constructor(baseUrl: string, apiKey: string, isDebug?: boolean) {
+		super(baseUrl, apiKey, isDebug);
 	}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)